docs/sources/mimir/manage/run-production-environment/production-tips/index.md

```diff
@@ -121,6 +121,21 @@ For these reasons, run the store-gateways on disks such as SSDs that have fast d
 The compactor requires a lot of disk space to download source blocks from the long-term storage and temporarily store the compacted block before uploading it to the storage.
 For more information about required disk space, refer to [Compactor disk utilization](../../../references/architecture/components/compactor/#compactor-disk-utilization).
 
+### Manage capacity for large tenants
+
+When working with large tenants, there are two compactor-specific settings to consider when planning or adjusting capacity:
+
+`-compactor.split-groups`
+`-compactor.split-and-merge-shards`
+
+For more information about how they work, refer to [Compaction algorithm]({{< relref "../architecture/components/compactor/index.md#compaction-algorithm" >}}).
+
+As a best practice, use 1 shard per every 8 million active series in a tenant.
```

Comment:
Is this value correct? Looking at https://github.com/grafana/mimir/pull/4621#discussion_r1152927182, it might be per every 25 million active series in a tenant. CC: @pstibrany .